pkg/api/server/register_containers.go

```diff
@@ -10,42 +10,879 @@ import (
+	//  - in: query
+	//    name: since
+	//    type:  string
+	//    description: needs description
+	//  - in: query
+	//    name: until
+	//    type:  string
+	//    description: needs description
+	//  - in: query
+	//    name: timestamps
+	//    type: bool
+	//    description: needs description
+	//  - in: query
+	//    name: tail
+	//    type: string
+	//    description: needs description
+	// produces:
+	// - application/json
+	// responses:
+	//   '200':
+	//     description: no error
+	//     schema:
+	//   '404':
+	//     description: no such container
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '500':
+	//     description: unexpected error
+	//     schema:
+	//      "$ref": "#/types/ErrorModel"
 	r.HandleFunc(VersionedPath("/containers/{name:..*}/logs"), APIHandler(s.Context, generic.LogsFromContainer)).Methods(http.MethodGet)
+	// swagger:operation POST /containers/{nameOrID}/pause containers pauseContainer
+	//
+	// Pause Container
+	//
+	// ---
+	// parameters:
+	//  - in: path
+	//    name: nameOrID
+	//    required: true
+	//    description: the name or ID of the container
+	// produces:
+	// - application/json
+	// responses:
+	//   '204':
+	//     description: no error
+	//     schema:
+	//   '404':
+	//     description: no such container
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '500':
+	//     description: unexpected error
+	//     schema:
+	//      "$ref": "#/types/ErrorModel"
 	r.HandleFunc(VersionedPath("/containers/{name:..*}/pause"), APIHandler(s.Context, handlers.PauseContainer)).Methods(http.MethodPost)
 	r.HandleFunc(VersionedPath("/containers/{name:..*}/rename"), APIHandler(s.Context, handlers.UnsupportedHandler)).Methods(http.MethodPost)
+	// swagger:operation POST /containers/{nameOrID}/restart containers restartContainer
+	//
+	// Restart Container
+	//
+	// ---
+	// parameters:
+	//  - in: path
+	//    name: nameOrID
+	//    required: true
+	//    description: the name or ID of the container
+	//  - in: query
+	//    name: t
+	//    type: int
+	//    description: timeout before sending kill signal to container
+	// produces:
+	// - application/json
+	// responses:
+	//   '204':
+	//     description: no error
+	//     schema:
+	//   '404':
+	//     description: no such container
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '500':
+	//     description: unexpected error
+	//     schema:
+	//      "$ref": "#/types/ErrorModel"
 	r.HandleFunc(VersionedPath("/containers/{name:..*}/restart"), APIHandler(s.Context, handlers.RestartContainer)).Methods(http.MethodPost)
+	// swagger:operation POST /containers/{nameOrID}/start containers startContainer
+	//
+	// Start a container
+	//
+	// ---
+	// parameters:
+	//  - in: path
+	//    name: nameOrID
+	//    required: true
+	//    description: the name or ID of the container
+	//  - in: query
+	//    name: detachKeys
+	//    type: string
+	//    description: needs description
+	// produces:
```

Comment:
Need to research what produces should include for empty body.

Reply:
agree, for next pr